src/styles/card.scss

```
@@ -0,0 +1,101 @@
+@import "variables";
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: block;
+  transition-duration: 300ms;
+
+  &__pic {
+   display: block;
+   margin: 0 auto;
+   height: 134px;
+   width: 160px;
+  }
+
+  &__name {
+    color: $text-color;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 40px;
+    margin-bottom: 0;
+    line-height: 18px;
+  }
+
+  &__code {
+    font-weight: 400;
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 140%;
+    margin-top: 4px;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    color: $text-color;
+    font-size: 10px;
+    margin-top: 3px;
+    text-align: center;
+  }
+
+  &__reviews-wrap{
+    height: 16px;
+  }
+
+  &__reviews-wrap,
+  &__price-wrap {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price {
+    font-weight: 400;
+    color: $secondary-color;
+    font-size: 12px;
+    line-height: 150%;
+    margin-top: 24px;
+    margin-bottom: 16px;
+    display: inline-block;
+
+    &_sum {
+      color: $text-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+      margin-top: 24px;
+      margin-bottom: 16px;
+      display: inline-block;
+    }
+  }
+
+  &__buy {
+    padding: 11px;
+    background-color: $blue-color;
+    border: 1px solid $blue-color;
+    border-radius: 5px;
+    display: block;
+
+    color: $white-color;
+    text-decoration: none;
+    text-transform: uppercase;
+    text-align: center;
+    line-height: 16px;
+    font-weight: 700;
+    font-size: 14px;
+    transition-duration: 300ms;
+  }
+
+  &__buy:hover {
+    background-color: $white-color;
+    color: $blue-color;
+  }
```

```suggestion
 &:hover {
      background-color: $white-color;
      color: $blue-color;
    } 
  }
```
make use of sass nesting